src/main/java/org/fcrepo/auth/webac/WebACAuthorizationDelegate.java

```diff
@@ -15,6 +15,17 @@
  */
 package org.fcrepo.auth.webac;
 
+// The WEBAC_HAS_ACL variable does not exist (we don't even have a namespace for it yet).
+import static org.fcrepo.auth.webac.URIConstants.WEBAC_HAS_ACL;
```

Comment:
Could we use acl:accessControl for this? From the ontology
```
<rdf:Property rdf:about="http://www.w3.org/ns/auth/acl#accessControl">
    <comment>The Access Control file for this information resource.
    This may of course be a virtual resorce implemented by the access control system.
    Note also HTTP's header  Link:  foo.meta ;rel=meta can be used for this.</comment>
    <domain rdf:resource="http://www.w3.org/2006/gen/ont#InformationResource"/>
    <label>access control</label>
    <range rdf:resource="http://www.w3.org/2006/gen/ont#InformationResource"/>
    <subPropertyOf rdf:resource="http://www.w3.org/2000/01/rdf-schema#seeAlso"/>
</rdf:Property>
```